src/index.js

```diff
@@ -21,6 +21,9 @@ const updateExp = require('./modules/updateExp');
 
 // commands initializers
 const pong = require('./modules/pong.command');
+const poll = require('./modules/poll.command');
+const vote = require('./modules/vote.command');
+const close = require('./modules/close.command');
```

Comment:
expose ONE poll module, and inside keep all three commands... and keep close/vote/poll inside "poll" folder, ok?

Reply:
OK